recipes/plumpy/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "plumpy" %}
+{% set version = "0.7.9"%}
+{% set sha256 = "a14c20c61c086414895142d22d4ba9b4a5267a743dddfff860d947a5126fdb79" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-fixed.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-fixed.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python 2.7*
+    - setuptools
+
+  run:
+    - python 2.7*
+    - frozendict
+    - enum34
```

Comment:
Add selectors to `enum34` and `futures` based on:

https://bitbucket.org/aiida_team/plum/src/5d23c0e97b8fc12a0a79cf29a516ab66ffe89c5a/setup.py?at=master&fileviewer=file-view-default#setup.py-35:36

see: https://conda.io/docs/building/meta-yaml.html#preprocessing-selectors